app/src/main/java/com/sdp/movemeet/chat/ChatActivity.java

```diff
@@ -49,35 +57,30 @@
 
     private static final String TAG = "ChatActivity";
 
-    public static final String MESSAGE_CHILD = "messages";
-    public static final String ANONYMOUS_NAME = "anonymous_name";
-    public static final String ANONYMOUS_ID = "anonymous_id";
-    public static final int DEFAULT_MSG_LENGTH_LIMIT = 10;
+    public static final String CHATS_CHILD = "chats";
+    public static String ROOM_CHILD = "general_chat";
```

Comment:
There is still a general chat ? Or it's for test ?

Reply:
The general chat still exists yes, it is currently accessible from the side nav panel created last week by JL! In addition, now each activity has its own dedicated chat.